modules/rest-api/pages/rest-xdcr-adv-settings.adoc

```diff
@@ -320,68 +382,118 @@ It cannot be set when an individual replication is created, or when its settings
 | Default: 4.
 The maximum number of threads used per node, to support XDCR.
 A greater number of threads increases parallelism, and may thereby produce enhanced XDCR performance.
+
 This setting is only returned when global settings are retrieved; and can only be set when global settings are set.
 It cannot be set when an individual replication is created, or when its settings are modified.
 
+| `hlvPruningWindowSec`
+| Integer
+| Used in conflict-resolution. Developer Preview only.
+
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `jsFunctionTimeoutMs`
+| Integer
+| Used in conflict-resolution. Developer Preview only.
+
+This setting can be established and retrieved either for an individual replication or globally.
+
 | `logLevel`
 | String
 | Default: `Info`.
 The level of logging, such as `Error`/`Info`/`Debug`/`Trace`.
 
+This setting can be established and retrieved either for an individual replication or globally.
+
+| `mergeFunctionMapping`
+| JSON Object
+|
```

Comment:
Used in *custom* conflict-resolution. Developer Preview only.

Reply:
We want to be able to use different merge functions for different collections. The JSON obj should be a map of string KVs that contains collection specifier (scope.collection) to merge function name

Reply:
I have written: This is available in Developer Preview Mode only. 
Each of the key-value pairs in the JSON object should map a collection specifier (`scope.collection`) to the name of a defined merge-function.